### ISSUE TITLE
export default Layout; (with semicolon)

### ISSUE THREAD

**Issue by pomber** [mdx-js/mdx#277]
Hi, I noticed that the mdx examples don't use semicolons, are they valid?

Because `export default Layout;` is failing: 

```js
mdx.sync(`
import Layout from './Layout';

export default Layout;

# Hello, world!
`)
```

returns:
```jsx
import Layout from './Layout';

export default ({components, ...props}) => <MDXTag name="wrapper" Layout={Layout;} layoutProps={props} components={components}>

<MDXTag name="h1" components={components}>{`Hello, world!`}</MDXTag></MDXTag>
```

(see `Layout={Layout;}`)

**Comment by silvenon** [comment]
We should either start supporting semicolons or explicitly prohibit them.

**Comment by timneutkens** [comment]
In feel like it should be supported 👍

**Comment by peduarte** [comment]
Hey there 👋 

I just spent a couple of hours trying to understand why my default export wrapper wasn't working.

Turns out it was also because I was using semi colon.

I agree with @silvenon that semi colons should be either supported or prohibited.

I'd personally prefer if it was supported 👍

**Comment by silvenon** [comment]
Me too! Prohibiting them would be such a confusing limitation. I'm currently a bit burned out from solving deeper parsing issues, so it will take me awhile to add support for this, unless someone else beats me to it. 😉

**Comment by peduarte** [comment]
Totally understandable! No pressure. I'm not sure I can help add support to this, it's probably above me. But I can update the docs with a *note*!

**Comment by silvenon** [comment]
Released in [v0.15.6](https://github.com/mdx-js/mdx/releases/tag/v0.15.6)! It turned out to be an easy fix.
